src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -96,56 +43,86 @@ export const TradesGrid: React.FC = () => {
-            trades.map((trade) => (
-              <TableBodyRow
-                key={trade.tradeId}
-                highlight={trade.tradeId === highlightedRow}
-                onClick={() => tryBroadcastContext(trade.symbol)}
-              >
-                <StatusIndicator
-                  status={trade.status}
-                  aria-label={trade.status}
-                />
-                {colFields.map((field, i) => (
-                  <TableBodyCell
-                    key={field}
-                    numeric={
-                      colConfigs[field].filterType === "number" &&
-                      field !== "tradeId"
-                    }
-                    rejected={trade.status === "Rejected"}
-                  >
-                    {colConfigs[field].valueFormatter?.(trade[field]) ??
-                      trade[field]}
-                  </TableBodyCell>
+    <TableBodyRow
+      key={trade.tradeId}
+      highlight={trade.tradeId === highlightedRow}
+      onClick={() => tryBroadcastContext(trade.symbol)}
+    >
+      <StatusIndicator status={trade.status} aria-label={trade.status} />
+      <Spacer aria-hidden={true} />
+      {colFields.map((field, i) => (
+        <TableBodyCell
+          key={field}
+          numeric={
+            colConfigs[field].filterType === "number" && field !== "tradeId"
+          }
+          rejected={trade.status === "Rejected"}
+          relativeWidth={colConfigs[field].width}
+        >
+          {colConfigs[field].valueFormatter?.(trade[field]) ?? trade[field]}
+        </TableBodyCell>
+      ))}
+    </TableBodyRow>
+  )
+}
+
+export const TradesGrid: React.FC<any> = ({ height = 200, width = 200 }) => {
+  const trades = useTableTrades()
+  const highlightedRow = useTradeRowHighlight()
+  const ref = useRef(null)
+
+  return (
+    <VirtuosoTableWrapper>
+      <VirtuosoTable role="grid">
+        <TableWrapper relativeWidth={width}>
+          <Table>
+            <caption id="trades-table-heading" className="visually-hidden">
+              Reactive Trader FX Trades Table
+            </caption>
+            <TableHead>
+              <TableHeadRow>
+                <StatusIndicatorSpacer scope="col" aria-label="Trade Status" />
+                <SpacerHeader aria-hidden={true} />
+                {colFields.map((field) => (
+                  <TableHeadCellContainer key={field} field={field} />
                 ))}
-              </TableBodyRow>
-            ))
-          ) : (
-            <TableBodyRow>
-              <StatusIndicatorSpacer aria-hidden={true} />
-              <TableBodyCell colSpan={colFields.length}>
-                No trades to show
-              </TableBodyCell>
-            </TableBodyRow>
+              </TableHeadRow>
+            </TableHead>
+          </Table>
+          {trades.length !== 0 && (
+            <Virtuoso
+              ref={ref}
+              style={{
+                height: height - 120 + "px",
```

Comment:
Where does the 120 come from? Can it be calculated? Or can we add a comment to explain?

Reply:
Added the comment, it's the sum of the rest of the elements within the Trades component